### PR TITLE
feat: send get request to improve update frequency of some parameters

### PR DESCRIPTION
**Pull request by jhonasiv**
Depends on:
- [ ] https://github.com/tidewise/drivers-deep_trekker/pull/18

### CODE REVIEW

File: deep_trekker.orogen [hunk 1]
```diff
@@ -24,6 +24,10 @@ task_context "RevolutionTask" do
     # Constant offset command for vertical thrusters to counteract buoyancy,
     # as buoyancy is positive in z, it should should be a negative value
     property "buoyancy_compensation_offset_command", "double"
+    # How often should a request be sent to update the powered reel states
+    property "powered_reel_update_interval", "base/Time"
+    # How often should a request be sent to update the revolution pose z attitude
+    property "pose_z_attitude_update_interval", "base/Time"
```
Comment (doudou) [inline deep_trekker.orogen:30]: Oh. That's going to improve depth and yaw control

**Review by doudou — APPROVED** [review]

**Comment by jhonasiv** [comment]
Pushed main branch instead of my local changes and it closed the PR...

File: tasks/RevolutionTask.cpp [hunk 2]
```diff
@@ -279,7 +312,10 @@ void RevolutionTask::receiveDeviceStateInfo()
         tryParseAndWriteIgnoringExceptions(
             [&]() { _camera_head_tilt_states.write(getCameraHeadTiltMotorState()); });
         tryParseAndWriteIgnoringExceptions([&]() {
-            _camera_head_tilt_states_rbs.write(getCameraHeadTiltMotorStateRBS());
+            auto camera_head_rbs = getCameraHeadTiltMotorStateRBS();
+            camera_head_rbs = "camera_pre";
+            camera_head_rbs = "camera_post";
```
Comment (kapeps) [inline tasks/RevolutionTask.cpp]: Did you forget to add .sourceFrame and .targetFrame?

Comment (jhonasiv) [reply tasks/RevolutionTask.cpp]: No, I forgot to remove these from this PR. The right one was already merged.

**Review by kapeps — APPROVED** [review]

**Pull request merged**